src/main/resources/application.yml

```diff
@@ -5,10 +5,16 @@ spring:
     cache-names: users
     caffeine:
       spec: expireAfterWrite=2m
-
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: http://localhost:9090/auth/issuer
```

Comment:
Do you have an external config file to override this when it's run in test/prod?